### PR TITLE
Wordy : sync with problem-specifications

### PR DESCRIPTION
**Pull request by Grociu**
Part of #1762

Changed function name to answer in test cases, exercise stub, and example.py to conform with canonical-data.json.
Changed test function names to fit the ones in canonical data.

### CODE REVIEW

**Comment by Grociu** [comment]
Hey, my first commit. I think I might be doing something wrong as I get an error from travis-ci.

File: exercises/wordy/example.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
                     "multiplied by": mul, "divided by": div}
 
 
-def calculate(stmt):
+def answer(stmt):
```
Comment (cmccandless) [inline exercises/wordy/example.py]:
```suggestion
def answer(question):
```

File: exercises/wordy/example.py [hunk 2]
```diff
@@ -11,7 +11,7 @@
...
     if not (stmt.startswith("What is ") and stmt.endswith("?")):
```
Comment (Grociu) [inline exercises/wordy/example.py]:
```suggestion
    if not (question.startswith("What is ") and question.endswith("?")):
```

Comment (Grociu) [inline exercises/wordy/example.py]:
```suggestion
    if not (question.startswith("What is ") and question.endswith("?")):
```

File: exercises/wordy/example.py [hunk 3]
```diff
@@ -11,7 +11,7 @@
...
         raise ValueError("Ill-formed question")
     words = stmt[8:-1].strip().lower().split()
```
Comment (Grociu) [inline exercises/wordy/example.py]:
```suggestion
    words = question[8:-1].strip().lower().split()
```

**Comment by cmccandless** [comment]
Merged; thanks for working on this!

**Pull request merged**